lib/active_admin/views/components/tabs.rb

```diff
@@ -15,17 +15,26 @@ def build(&block)
       end
 
       def build_menu_item(title, options, &block)
-        fragment = options.fetch(:id, title.parameterize)
+        fragment = options.fetch(:id, fragmentize(title))
+
         html_options = options.fetch(:html_options, {})
         li html_options do
           link_to title, "##{fragment}"
         end
       end
 
       def build_content_item(title, options, &block)
-        options = options.reverse_merge(id: title.parameterize)
+        options = options.reverse_merge(id: fragmentize(title))
         div(options, &block)
       end
+
+      private
+
+      def fragmentize(string)
+        result = string.parameterize
+        result = string.hash if result.blank?
```

Comment:
This is not safe! `string.hash` returns different values in each process.
```
➜ ruby -e 'puts "string".hash'
-1157288931546009962
➜ ruby -e 'puts "string".hash'
3581747432009468955
```
This means that the tabs didn't work across different server instances!

Reply:
Is this `id` used in the server side? I thought we just needed it to be unique per page and play nice with our js.

Reply:
```suggestion
        result = Digest::MD5.hexdigest(string) if result.blank?
```

Reply:
yes for the js it will work, but if you reload the page it will maybe break, depending on which server or which process your hit with the request.

Reply:
I don't see how it could break, but I'm fine with using digests too :+1:. It might help with caching, maybe, and I guess it gives more uniqueness guarantees, which in the unlikely case of collisions, would indeed break navigation.

Reply:
I saw https://github.com/activeadmin/activeadmin/pull/5650#discussion_r now, nice catch!